source/turtles/turtle_main.py

```diff
@@ -0,0 +1,79 @@
+import pygame
+from source.turtles.turtle_hero import Turtle, TurtleHero
+
+SCREENWIDTH = 800
+SCREENHEIGHT = 600
+GREEN = (20, 255, 140)
+
+size = (SCREENWIDTH, SCREENHEIGHT)
+screen = pygame.display.set_mode(size)
+pygame.display.set_caption("Turtle Game")
+
+playerTurtle = TurtleHero("normal", 0.5, "Karol1", (200, 200))
+
+# Allowing the user to close the window...
+carryOn = True
+clock = pygame.time.Clock()
+
+all_sprites_list = pygame.sprite.Group()
+all_sprites_list.add(playerTurtle)
+
+while carryOn:
+    for event in pygame.event.get():
+        keys2 = pygame.key.get_pressed()
+        if event.type == pygame.QUIT:
+            carryOn = False
+        elif event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_SPACE and playerTurtle.is_jumping == 0:
+                playerTurtle.init_jump(400, 800)
+            if event.key == pygame.K_RIGHT and not keys2[pygame.K_RCTRL]:
```

Comment:
może daloby sie zagniezdzic bardziej te IFY? np jesli K_RCTRL, to (podIFY) itp

Reply:
Jeszcze warto byloby z tego zrobic metode w zolwiu, zeby te ify tutaj tak nie lataly luzem (ale to nie jest koneizne

Reply:
To potem, bo to nie jest docelowe sterowanie, wiem o tym